cmd/kubeadm/app/features/features.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/pkg/errors"
 	"k8s.io/apimachinery/pkg/util/version"
 	"k8s.io/component-base/featuregate"
+	kubefeatures "k8s.io/kubernetes/pkg/features"
```

Comment:
we cannot import this package in kubeadm as right now there is ongoing work to decouple kubeadm from `kuberenetes/pkg`.

Reply:
+1 on this see kubernetes/kubeadm#1600